### PR TITLE
Adding info about soft deletes, per issue #3167

### PR DESCRIPTION
**Pull request by lleonard-msft**
Updating descriptions, per #3617 

@dkershaw10 - Please review.

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[f31cb62](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4041/commits/f31cb62d22ef9996623d129f69773424c748ff9d)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[f31cb62](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4041/commits/f31cb62d22ef9996623d129f69773424c748ff9d)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 3 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[f31cb62](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4041/commits/f31cb62d22ef9996623d129f69773424c748ff9d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/group-delete.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/soft-delete-desc/api-reference/beta/api/group-delete.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/user-delete.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/soft-delete-desc/api-reference/beta/api/user-delete.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/group-delete.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/soft-delete-desc/api-reference/v1.0/api/group-delete.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/user-delete.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/soft-delete-desc/api-reference/v1.0/api/user-delete.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C25%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902251211578522-4041%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=XP51nxCxhYr2WXIqQiGyWOkaPHhu2s%2BVoz6LfS8sIcs%3D&st=2019-02-25T12%3A10%3A42Z&se=2019-03-28T12%3A15%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/beta/api/group-delete.md [hunk 1]
```diff
@@ -10,11 +10,12 @@ ms.prod: "groups"
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
-Deletes a group.
+Deletes a group.  
 
-When a group is deleted, the item is added to [deleted items](../resources/directory.md). The group will remain in deleted items for up to 30 days. A group can be fully restored from deleted items during the 30 days. After 30 days, deleted items are permanently deleted.
+When deleted, Office 365 groups to a temporary container and can be restored within 30 days.  After that time, they are permanently deleted.  To learn more, see [deletedItems](../resources/directory.md).  This applies only to Office 365 groups.
```
Comment (dkershaw10) [inline api-reference/beta/api/group-delete.md]: Grammar - missing word?  Not sure why we are changing this text, but it's fine by me either way.

File: api-reference/v1.0/api/group-delete.md [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
 localization_priority: Normal
 ms.prod: "groups"
 ---
 
 # Delete group
-Delete group.
+
+Delete group.  
+
+When deleted, Office 365 groups to a temporary container and can be restored within 30 days.  After that time, they are permanently deleted.  To learn more, see [deletedItems](../resources/directory.md).  This applies only to Office 365 groups.
```
Comment (dkershaw10) [inline api-reference/v1.0/api/group-delete.md]: Same - missing "are moved"?

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[9c37b75](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4041/commits/9c37b75868afb7a52f12cc9ab6e95e34967a8052)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

File: api-reference/beta/api/group-delete.md (hunk 1 above)
Comment (lleonard-msft) [reply api-reference/beta/api/group-delete.md]: Oops. - Fixed.

File: api-reference/v1.0/api/group-delete.md (hunk 2 above)
Comment (lleonard-msft) [reply api-reference/v1.0/api/group-delete.md]: Oops - Fixed.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[9c37b75](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4041/commits/9c37b75868afb7a52f12cc9ab6e95e34967a8052)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 3 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[9c37b75](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4041/commits/9c37b75868afb7a52f12cc9ab6e95e34967a8052)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/group-delete.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/soft-delete-desc/api-reference/beta/api/group-delete.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/user-delete.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/soft-delete-desc/api-reference/beta/api/user-delete.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/group-delete.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/soft-delete-desc/api-reference/v1.0/api/group-delete.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/user-delete.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/soft-delete-desc/api-reference/v1.0/api/user-delete.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C25%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902251808597903-4041%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=zFXnLQTePdP4dJc9BKzkRNw2a8Kimh1RiJxtH2V85Ug%3D&st=2019-02-25T18%3A07%3A45Z&se=2019-03-28T18%3A12%3A45Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by lleonard-msft** [comment]
Minor issues fixed; #sign-off

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[77a680e](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4041/commits/77a680e8831600234a747f89add83189c703d66f)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[77a680e](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4041/commits/77a680e8831600234a747f89add83189c703d66f)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 3 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[77a680e](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4041/commits/77a680e8831600234a747f89add83189c703d66f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/group-delete.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/soft-delete-desc/api-reference/beta/api/group-delete.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/user-delete.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/soft-delete-desc/api-reference/beta/api/user-delete.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/group-delete.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/soft-delete-desc/api-reference/v1.0/api/group-delete.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/user-delete.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/soft-delete-desc/api-reference/v1.0/api/user-delete.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C25%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902252331300857-4041%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=bCB0cYoYKs6u55ktIrRtbTg2ig5NqI7IT8Eik1TWeFE%3D&st=2019-02-25T23%3A30%3A03Z&se=2019-03-28T23%3A35%3A03Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**